modules/virt-configuring-sriov-networks-for-vms.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-defining-an-sriov-network.adoc
+
+// This is similar to the OpenShift `configuring-sr-iov-networks` module,
+// modified for virt context
+
+// Because of an existing issue in go-yaml, the strings 'on' and 'off'
+// are interpreted as booleans, not strings. The SR-IOV admission controller
+// will reject 'spoofCheck' and 'trust' if the values are not strings.
+// So these values must be explicitly quoted in the YAML.
+// https://github.com/go-yaml/yaml/issues/214
+
+[id="virt-configuring-sr-iov-networks-for-vms_{context}"]
+= Configuring SR-IOV additional network for virtual machines
+
+You can configure an additional network that uses SR-IOV hardware by creating a SriovNetwork Custom Resource (CR).
+When you create a SriovNetwork CR, the SR-IOV Operator automatically creates a NetworkAttachmentDefinition CR.
+
+[NOTE]
+=====
+Do not modify or delete a SriovNetwork Custom Resource (CR) if it is attached to any Pods in the `running` state.
+=====
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the following SriovNetwork CR, and then save the YAML in the `<name>-sriov-network.yaml` file. Replace `<name>` with a name for this additional network.
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: <name> <1>
+  namespace: sriov-network-operator <2>
+spec:
+  ipam: <ipam> <3>
+  networkNamespace: <target_namespace> <4>
+  resourceName: <sriov_resource_name> <5>
+  spoofChk: "off" <6>
+----
+<1> Replace `<name>` with a name for the CR. The Operator will create a NetworkAttachmentDefinition CR with same name.
+<2> Specify the namespace where the SR-IOV Operator is installed.
+<3> Optional: Replace `<ipam>` a configuration object for the ipam CNI plug-in as a YAML block scalar. The plug-in manages IP address assignment for the attachment definition.
+<4> Optional: Replace `<target_namespace>` with the namespace where the NetworkAttachmentDefinition CR will be created. The default value is `openshift-sriov-network-operator`.
+<5> Replace `<sriov_resource_name>` with the value for the `.spec.resourceName` parameter from the SriovNetworkNodePolicy CR that defines the SR-IOV hardware for this additional network.
+<6> Optional: Replace `<spoof_check>` with the spoof check mode of the VF. The allowed values are the strings `"on"` and `"off"`.
++
+[IMPORTANT]
+====
+You must enclose the value you specify in quotes or the CR will be rejected by the SR-IOV Network Operator.
```

Comment:
Which value? Spoof check?

Reply:
Yes, this value is a string, not a boolean. (So it can actually be on, off, or unset; I forget the distinction.)